master/reference/felix/configuration.md

```diff
@@ -86,6 +86,7 @@ The Kubernetes API datastore driver reads its configuration from Kubernetes-prov
 | `IptablesRefreshInterval`            | `FELIX_IPTABLESREFRESHINTERVAL`            | Period, in seconds, at which Felix re-checks all iptables state to ensure that no other process has accidentally broken {{site.prodname}}'s rules. Set to 0 to disable iptables refresh. [Default: `90`] | int |
 | `LogPrefix`                          | `FELIX_LOGPREFIX`                          | The log prefix that Felix uses when rendering LOG rules. [Default: `calico-packet`] | string |
 | `MaxIpsetSize`                       | `FELIX_MAXIPSETSIZE`                       | Maximum size for the ipsets used by Felix to implement tags. Should be set to a number that is greater than the maximum number of IP addresses that are ever expected in a tag. [Default: `1048576`] | int |
+| `NATPortRange`                       | `FELIX_NATPORTRANGE`                       | Port range used by iptables for port mapping when doing outgoing NAT.  | string |
```

Comment:
Please can you include the default and an example of the syntax?

Reply:
The default is a little bit involved explain: It maps ports below 512 to another port below 512, ports below 512-1024 to another port in that range and for everything above 1024 it picks a random port above 1024. Should I try to summarize that in this table?